boot/zephyr/nrf_cleanup.c

```diff
@@ -51,13 +48,6 @@ static inline void nrf_cleanup_rtc(NRF_RTC_Type * rtc_reg)
 }
 #endif
 
-#if defined(CONFIG_NRF_GRTC_TIMER)
-static inline void nrf_cleanup_grtc(void)
-{
-    nrfx_grtc_uninit();
```

Comment:
I don't follow this, the function disables an interrupt, clears up assigned channels and stops the sys counter, if MCUboot has set these up then why should they not be cleaned up before booting an application which has it's own init functions for setting these features up in the configuration it wants (which might be completely different)? @bjarki-andreasen @nordic-krch @adamkondraciuk

Reply:
I see you've already mentioned it in one of the conversations, missed this